components/prism-css.js

```diff
@@ -7,7 +7,12 @@
 		'atrule': {
 			pattern: /@[\w-]+[\s\S]*?(?:;|(?=\s*\{))/,
 			inside: {
-				'rule': /@[\w-]+/
+				'rule': /^@[\w-]+/,
+				'selector-function-argument': {
+					pattern: /(\bselector\s*\((?!\s*\))\s*)(?:[^()]|\((?:[^()]|\([^()]*\))*\))+?(?=\s*\))/,
+					lookbehind: true,
+					alias: 'selector'
```

Comment:
Should this be aliased to `argument`? Do we have a token for that?

Reply:
> Do we have a token for that?

No. Neither `argument` nor something like `parameter`.

Also, I initially wanted to call this just `selector` but this wasn't possible because of the `rest` property which includes CSS, including the actual `selector` token.